.clang-format

```diff
@@ -2,3 +2,4 @@ BasedOnStyle:  Google
 AccessModifierOffset: -2
 ColumnLimit: 200
 IndentWidth: 4
+AllowShortEnumsOnASingleLine: false
```

Comment:
Without this, it tried to put all my enum values on a single line.

Reply:
Huh ok, plz push this change to the overview repo

Reply:
https://github.com/UCI-Rocket-Project/rocket2-overview/pull/1